Documentation/ceph-csi-drivers.md

```diff
@@ -295,6 +317,42 @@ SNAPID NAME
      4 csi-rbd-pvc-c20495c0d5de11e8-snap-4c0b455b-d5fe-11e8-bebb-525400123456 1024 MB Mon Oct 22 13:28:03 2018
 ```
 
+#### Restore the snapshot to a new PVC
+
+```yaml
```

Comment:
could we include these yamls in the example folder? Then users don't actually have to copy-paste.